### PR TITLE
gh-864 - Finished off refactoring

### PR DESCRIPTION
**Pull request by m607123**
Refactored any remaining toString, hashCode and equals methods to the ACL3 style.

### CODE REVIEW

**Comment by m607123** [comment]
Closing while errors are fixed.

File: core/data/src/main/java/uk/gov/gchq/gaffer/data/element/Element.java [hunk 1]
```diff
@@ -110,9 +110,19 @@ public boolean shallowEquals(final Element element) {
...
+        }
+
+        final Element element = (Element) obj;
+
+        return new EqualsBuilder()
+                .append(properties, element.properties)
+                .isEquals();
     }
 
     public boolean equals(final Element element) {
```
Comment (t616178) [inline core/data/src/main/java/uk/gov/gchq/gaffer/data/element/Element.java]: Is this method used any more?

File: core/data/src/main/java/uk/gov/gchq/gaffer/data/element/LazyProperties.java [hunk 2]
```diff
@@ -153,21 +153,29 @@ public boolean containsValue(final Object value) {
...
-    public boolean equals(final Object o) {
-        return properties.equals(o);
-    }
+    public boolean equals(final Object obj) {
+        return properties.equals(obj);
 
-    @Override
-    public int hashCode() {
-        return properties.hashCode();
+//        if (this == obj) {
```
Comment (t616178) [inline core/data/src/main/java/uk/gov/gchq/gaffer/data/element/LazyProperties.java]: Could you check this commented block?

File: core/operation/src/main/java/uk/gov/gchq/gaffer/operation/data/EntitySeed.java [hunk 3]
```diff
@@ -49,23 +51,27 @@ public boolean equals(final Object o) {
...
 
         final EntitySeed that = (EntitySeed) o;
-        return !(vertex != null ? !vertex.equals(that.getVertex()) : that.getVertex() != null);
+
+        return new EqualsBuilder()
+                .append(vertex, that.vertex)
+                .isEquals();
     }
 
+    // Important not to alter hashcode implementation for.. some reason
```
Comment (t616178) [inline core/operation/src/main/java/uk/gov/gchq/gaffer/operation/data/EntitySeed.java]: Could you be a little more specific?

File: core/operation/src/main/java/uk/gov/gchq/gaffer/operation/data/EntitySeed.java [hunk 4]
```diff
@@ -49,23 +51,27 @@ public boolean equals(final Object o) {
         if (this == o) {
```
Comment (t616178) [inline core/operation/src/main/java/uk/gov/gchq/gaffer/operation/data/EntitySeed.java]: The reference name here should be longer than one character - could you update this to:

```
public boolean equals(final Object obj) {
    // ...
}
``` 

Could you also update the other ```equals()``` methods too?

**Pull request merged**